components/formats-bsd/src/loci/formats/tools/PrintDomains.java

```diff
@@ -52,18 +52,13 @@ public static void main(String[] args) {
     // get a list of all available readers
     IFormatReader[] readers = new ImageReader().getReaders();
 
-    Hashtable<String, Vector<IFormatReader>> domains =
-      new Hashtable<String, Vector<IFormatReader>>();
-
-    for (String domain : FormatTools.ALL_DOMAINS) {
-      domains.put(domain, new Vector<IFormatReader>());
-    }
+    final TreeMultimap<String, String> domains = TreeMultimap.create();
```

Comment:
Why are we not filling the `domains` map with the values in `ALL_DOMAINS` as before?